Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "finality-grandpa"
-version = "0.5.0"
+version = "0.6.0"
```

Comment:
this is a backwards-compatible change, right? so we could have version = `0.5.1`